pkg/jx/cmd/import.go

```diff
@@ -258,12 +260,17 @@ func (options *ImportOptions) Run() error {
 				return err
 			}
 		}
+
+		if options.UseDefaultGit {
+		    userAuth= config.CurrentUser(server, true)
```

Comment:
You should use the `options.InCluster` instead of `true` .

Reply:
@ccojocar 

You mean options.CommonOptions.InCluster()?